addons/mail/static/src/js/services/mail_manager.js

```diff
@@ -1004,7 +1004,7 @@ var MailManager =  AbstractService.extend({
                 _t("%d Messages"),
                 this._outOfFocusUnreadMessageCounter
             );
-            this.trigger_up('set_title_part', {
+            this.env.bus.trigger('set_title_part', {
```

Comment:
Not that I'm against the change, but isn't the service trigger_up mechanism supposed to reach the "root" and properly forward to the env at this point?

Reply:
What would you suggest? To redirect all events that aren't "call_service" on the bus? There were a few (< 3 I'd say) usecases, so I went for this solution. The root only redirects "call_service" events to this.env.services. For the rest, I adapted the code (there is this usecase, and I think I replaced a getSession() by a direct import).

Reply:
It is fine as you did it, I was just looking to understand better and you provided those info now.